torch/csrc/DynamicTypes.cpp

```diff
@@ -79,8 +79,11 @@ THPLayout* getTHPLayout(at::Layout layout) {
   return thp_layout;
 }
 
-PyObject* createPyObject(const at::Storage& storage) {
-  if (storage.device_type() != at::DeviceType::Meta &&
+PyObject* createPyObject(
+    const at::Storage& storage,
+    bool always_create_python_storage) {
+  if (!always_create_python_storage &&
+      storage.device_type() != at::DeviceType::Meta &&
```

Comment:
If I understand correctly, you haven't "fixed" the safety problem whereby a binding to a null storage lets you easily segfault. It seems better if you fix that problem, and then you can just unconditionally allow for Python binding to storage.